.dockerignore

```diff
@@ -5,5 +5,6 @@ fastlane.log
 fastlane.verbose.log
 public/.dist/*
 **/node_modules/*
+node_modules
```

Comment:
My favorite folder isn't included 😢

Reply:
don't worry, you can run `npm install` and that will still download 85gb of `pad-left` dependencies.

Reply:
Beautiful